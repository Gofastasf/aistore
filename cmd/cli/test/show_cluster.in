// RUN remais
ais show cluster | grep -E "(Cluster|Endpoint|Proxies|Targets|Capacity|Cluster Map|Software|Backend|Deployment|Status|Rebalance|Authentication|Version|Build)"
ais show dashboard | grep -E "(Performance and Health|State|I/O Errors|Load Avg|Disk Usage|Network|Storage|Filesystems|Running Jobs|Cluster|Endpoint|Proxies|Targets|Capacity|Cluster Map|Software|Backend|Deployment|Status|Rebalance|Authentication|Version|Build)" | head -23
ais show remote dashboard | grep -E "(Performance and Health|State|I/O Errors|Load Avg|Disk Usage|Network|Storage|Filesystems|Running Jobs)"
ais show cluster --json // IGNORE
ais show cluster $RANDOM_TARGET // IGNORE
ais show cluster FOO // FAIL "expecting a valid NODE_ID or node type"
