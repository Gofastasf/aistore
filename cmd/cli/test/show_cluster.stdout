^Cluster:$
^.*Proxies:.*$
^.*Targets:.*$
^.*Capacity:.*$
^.*ClusterMap:.*$
^.*Software:.*$
^.*Backend:.*$
^.*Deployment:.*$
^.*Status:.*$
^.*Rebalance:.*$
^.*Authentication:.*$
^.*Version:.*$
^.*Build:.*$
^Performance and Health:$
^.*State:.*$
^.*I/O Errors:.*$
^.*Load Avg:.*$
^.*Disk Usage:.*$
^.*Network:.*$
^.*Storage:.*$
^.*Filesystems:.*$
^Cluster:$
^.*Proxies:.*$
^.*Targets:.*$
^.*Endpoint:.*$
^.*Capacity:.*$
^.*Cluster Map:.*$
^.*Software:.*$
^.*Backend:.*$
^.*Deployment:.*$
^.*Status:.*$
^.*Rebalance:.*$
^.*Authentication:.*$
^.*Version:.*$
^.*Build:.*$
